src/boundary_operators.jl

```diff
@@ -22,6 +22,35 @@ function rem1(idx,L)
 end
 
 
+#= LEAKY BCs =#
```

Comment:
Also, left and right should be split like the others. This is the only one which isn't following the format.